### PR TITLE
Added watchexec + cheat sheets

### PR DESCRIPTION
**Pull request by anuragsati**
## Description 
- Added `watchexec` 
- Added cheat sheets for `watchexec`

### CODE REVIEW

File: watchexec/install.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        mkdir -p "$(dirname $pkg_src_cmd)"
+
+        # mv ./watchexec-*/watchexec ~/.local/opt/watchexec-v0.99.9/bin/watchexec
+        mv ./watchexec-*/watchexec "$pkg_src_cmd"
+    }
+
+    # pkg_get_current_version is recommended, but (soon) not required
+    pkg_get_current_version() {
+        # 'watchexec --version' has output in this format:
+        #       watchexec 0.99.9 (rev abcdef0123)
```
Comment (coolaj86) [inline watchexec/install.sh:33]: Doesn't output git rev:
```bash
watchexec --version
watchexec 1.14.0
```

**Comment by coolaj86** [comment]
This is really good, but I'd like a **few of changes**:

1. Use bash code fences and don't include `$` (these should be copy/paste)
    ~~~md
    ```bash
    watchexec blah blah blah
    ```
    ~~~
2. We don't want to directly rip off the README of the project, but rather select a few use cases that you would use and put it in your own words (or you can leave it up to me to do this if you don't actually use `watchexec`) - either way I don't feel comfortable directly copying the original author's README.

I've got this in preview at https://beta.webinstall.dev/watchexec

I've tested on Mac, Linux, and Windows. 👍

**Comment by anuragsati** [comment]
Thanks for reviewing my code.
I'm extremely sorry for using original repository's readme, I don't use `watchexec`, it would be very helpful if you could update the readme for this.

**Comment by coolaj86** [comment]
Thanks. I'll merge this in as-is and I'll update the readme on my end.

**Review by coolaj86 — APPROVED** [review]

**Pull request merged**